doc/classes/ImmediateMesh.xml

```diff
@@ -4,9 +4,19 @@
 		Mesh optimized for creating geometry manually.
 	</brief_description>
 	<description>
-		Mesh optimized for creating geometry manually, similar to OpenGL1.x immediate mode.
+		A mesh type optimized for creating geometry manually, similar to OpenGL1.x immediate mode.
```

Comment:
I wonder if we can mention that this is similar to ImmediateGeometry from Godot 3. That would make this class show up in the docs search results if people are looking for the old node. Though I can understand if references to old equivalents aren't wanted in the docs.

Reply:
Any other opinions? We don't have this for any classes currently, so I haven't added it - but I could. Personally think it would be better to have some sort of mechanism for helping people find renamed classes, than to add it to the raw text.

Reply:
I'm fine with not adding it for now, I think this is worth more discussion and shouldn't block this PR

Reply:
Yeah we thought about adding a system to use the renames from the Godot 3 to 4 project converter to work as aliases when searching the docs.

Reply:
is that tracked somewhere so this can be added to the list? It's not really a rename though, it also changed from being it's own Node to being a resource for MeshInstances